tests/integration/conftest.py

```diff
@@ -9,14 +9,24 @@
 from nequip.utils.unittests.utils import _training_session, _check_and_print
 
 
+try:
+    import schedulefree  # noqa: F401
+
+    _SCHEDULEFREE_INSTALLED = True
+except ImportError:
+    _SCHEDULEFREE_INSTALLED = False
+
+
 # available training modules for integration tests
-# to save time, we only check the EMA ones for now
+# to save time, we only check the EMA and ScheduleFree ones for now
```

Comment:
for this same reason I am inclined to disable the schedulefree CI tests and not install schedulefree in the default CI once this is ready to merge